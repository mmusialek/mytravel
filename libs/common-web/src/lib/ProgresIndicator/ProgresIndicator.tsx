export const ProgresIndicator = () => {
  return (<div>Loading ...</div>)
}
