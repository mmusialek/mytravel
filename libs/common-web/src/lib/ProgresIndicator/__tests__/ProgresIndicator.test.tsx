import renderer from "react-test-renderer";
import { ProgresIndicator } from "../ProgresIndicator";

describe('ProgresIndicator', () => {
  it('should render', () => {
    const comp = renderer.create(<ProgresIndicator />).toJSON();
    expect(comp).toMatchSnapshot();
  });
});
