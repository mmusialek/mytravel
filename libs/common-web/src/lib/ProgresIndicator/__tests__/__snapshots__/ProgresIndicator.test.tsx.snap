// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgresIndicator should render 1`] = `
<div>
  Loading ...
</div>
`;
